tests/benchdnn/pool/pool.cpp

```diff
@@ -150,6 +151,15 @@ void skip_invalid_prb(const prb_t *prb, res_t *res) {
             return;
         }
     }
+
+#if DNNL_AARCH64_USE_ACL
+    // Since ACL supports only forward pass.
+    // Ref: https://github.com/oneapi-src/oneDNN/issues/1205
+    if (prb->dir & FLAG_BWD) {
```

Comment:
Does it make sense to skip all unimplemented cases when `DNNL_AARCH64` is defined?
https://github.com/oneapi-src/oneDNN/blob/acb63f4949c9bdf6952f673f55b3dd3102d771b0/tests/benchdnn/dnnl_common.hpp#L407-L421

@dzarukin

Reply:
This check has been moved into `skip_unimplemented_prb()`. An additional check has been added into `pool::init_pd` to accomodate this